server/jetty/src/test/java/io/deephaven/server/jetty/JettyFlightRoundTripTest.java

```diff
@@ -41,4 +46,22 @@ public interface JettyTestComponent extends TestComponent {
     protected TestComponent component() {
         return DaggerJettyFlightRoundTripTest_JettyTestComponent.create();
     }
+
+    @Test
+    public void jsPluginsManifest() throws Exception {
+        // TODO(deephaven-core#3004): Jetty HTTP testing
+        // Would be good to move this to testing setup with minimal overhead, where things can easily be injected and
+        // changed.
+        final HttpClient client = new HttpClient();
+        try {
+            client.start();
+            final ContentResponse response =
+                    client.GET(String.format("http://localhost:%d/js-plugins/manifest.json", port));
+            assertThat(response.getStatus()).isEqualTo(200);
+            assertThat(response.getContentAsString()).isEqualTo("{\"plugins\":[]}");
+            assertThat(response.getHeaders().contains("ETag")).isTrue();
```

Comment:
the current etag impl is counterproductive, should probably remove this assertion